code/controllers/subsystems/chemistry.dm

```diff
@@ -53,3 +56,28 @@ var/datum/controller/subsystem/chemistry/SSchemistry
 	src.active_holders = SSchemistry.active_holders
 	src.chemical_reactions = SSchemistry.chemical_reactions
 	src.chemical_reagents = SSchemistry.chemical_reagents
+
+/datum/controller/subsystem/chemistry/proc/load_secret_chemicals()
+	var/list/chemconfig = list()
+	try
+		chemconfig = json_decode(return_file_text("config/secretchem.json"))
+	catch(var/exception/e)
+		log_debug("SSChemistry: Warning: Could not load config, as secretchem.json is missing - [e]")
+		return
+
+	for (var/chemical in chemconfig["chemicals"])
+		log_debug("SSChemistry: Loading chemical: [chemical]")
+		var/datum/chemical_reaction/cc = new()
+		cc.name = chemconfig["chemicals"][chemical]["name"]
```

Comment:
Might be best to separate out the `chemconfig["chemicals"]` list into its own var, since it's the only one accessed and it looks kind of messy atm.